src/scripts/checkHeader.sh

```diff
@@ -22,24 +22,27 @@ FILES=$(echo "$FILES" | grep -v "src\/client\/app\/images\/.*")
 FILES=$(echo "$FILES" | grep -v "src\/client\/favicon\.ico")
 # Filter out test data
 FILES=$(echo "$FILES" | grep -v "src\/server\/test\/db\/data\/.*")
+# Filter out outside scripts
+FILES=$(echo "$FILES" | grep -v "src\/scripts\/oed\.service")
+FILES=$(echo "$FILES" | grep -v "src\/scripts\/OEDCron\.bash")
```

Comment:
This isn't really related to this PR, but I wonder if it would be nice to add something like `/* no MPL header */` to exclude a file from the MPL header check, rather than having to add them to this script.